### DISCUSSION TITLE
Ant Media Server 2.8.0 - Community and Enterprise Editions are available 🚀

### DISCUSSION THREAD

**Discussion by mekya** [ant-media/Ant-Media-Server#6061] [Announcements]
Hi Guys,
Ant Media Server 2.8.0 is available. You can get the community edition from [Ant Media Server's Github Releases](https://github.com/ant-media/Ant-Media-Server/releases/tag/ams-v2.8.0) and you can get the Enterprise Edition from [antmedia.io](https://antmedia.io)

**Highlights**
- Migrate to Java 17, Spring 6, Tomcat 10 and updates for almost all libraries 🏃 
- Improve Video Conferencing Performance and Stability 📲 
- Improve Stability by Benchmark Automations 💪 

Cheers 💯 
Oguz

**Comment by semobb** [comment]
How to install the latest community version of ant-media-server using docker one-click installation command?

**Reply by muratugureminoglu** [reply]
Hi @semobb 

The community version of Ant Media Server is not yet available on docker hub, however you can easily build and run it by following the documentation below[1].

[1] https://antmedia.io/docs/guides/clustering-and-scaling/docker/docker-and-docker-compose-installation/

Regards.

**Comment by devmatheus** [comment]
When do you plan to update the dockerhub? https://hub.docker.com/r/antmedia/enterprise/tags

**Reply by muratugureminoglu** [reply]
Hi @devmatheus 

We are planning this week to upload new image.

Regards.

**Reply by muratugureminoglu** [reply]
Hi @devmatheus 

I have prioritized it for you and 2.8.2 is available now. :)
